apps/twitch-bot/internal/command/command.go

```diff
@@ -34,78 +37,98 @@ func NewCommands(client *client.Clients, service service.Service, cooldownPeriod
 	}
 }
 
-func (c *commands) GetCommands() map[string]func(context context.Context, message twitch.PrivateMessage, commandName string, params []string) {
+func (c *commands) GetCommands() commandFuncReturn {
 	// TODO: command aliases
-	var commands = map[string]func(context context.Context, message twitch.PrivateMessage, commandName string, params []string){
-		"ping":      c.PingCommand,
-		"invite":    c.InviteCommand,
-		"senchabot": c.SenchabotCommand,
-		"sukru":     c.SukruCommand,
-		"sozluk":    c.SozlukCommand,
+	var commands = commandFuncReturn{
+		"ping":   c.PingCommand,
+		"invite": c.InviteCommand,
+		"sozluk": c.SozlukCommand,
 
 		"acmd": c.AddCommandCommand,
 		"ucmd": c.UpdateCommandCommand,
 		"dcmd": c.DeleteCommandCommand,
-		//"info": InfoCommandCommand,
 		"cmds": c.CmdsCommand,
 
 		"acmda": c.AddCommandAliasCommand,
 		"dcmda": c.DeleteCommandAliasCommand,
-		"help":  c.HelpCommand,
 
-		"kampus":       c.KampusCommand,
-		"frontendship": c.FrontendshipCommand,
+		"help": c.HelpCommand,
 	}
 
 	return commands
 }
 
-func (c *commands) RunStaticCommand(context context.Context, cmdName string, params []string, message twitch.PrivateMessage) {
-	cmds := c.GetCommands()
-
-	if cmd, ok := cmds[cmdName]; ok {
-		if c.isUserOnCooldown(message.User.Name) {
-			return
+func (c *commands) IsSystemCommand(commandName string) bool {
+	commandListMap := c.GetCommands()
+	for k := range commandListMap {
+		if k == commandName {
+			return true
 		}
```

Comment:
we don't need to iterate the map. we can directly check the existence of the key instead of iteration